chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +332,76 @@ int ChromeNetworkDelegate::OnBeforeURLRequest(
   return rv;
 }
 
+std::string generateBlockedElemhideJavaScript(std::string url, std::string relative_url) {
+  std::stringstream ss;
+
+  // basically copy-pasted JS and saved to resources from
+  // https://github.com/adblockplus/adblockpluschrome/blob/master/include.preload.js#L546
```

Comment:
i will create separate task for `core` to extract files to be used both in `core` and on mobile and separate (dependent) task for `adblockchromium` to checkout `core` and reuse js from it (without hardcoding to resources). so we will be able to call `gclient sync` and fetch `core` files and reuse them here.